demo/components-demo.tsx

```diff
@@ -29,6 +28,9 @@ export class ComponentsDemo extends React.Component<{}, {}>{
                 <TreeViewDemo />
             </div>
             <hr />
+            <h1>{'<DatePicker />'}</h1>
```

Comment:
Please adjust the demo section to the rest of the components. It's been changed meanwhile, methink.